drivers/bluetooth/hci/hci_ambiq.c

```diff
@@ -52,7 +52,7 @@ LOG_MODULE_REGISTER(bt_hci_driver);
 #define SPI_MAX_TX_MSG_LEN 524
 #define SPI_MAX_RX_MSG_LEN 258
 
-static uint8_t g_hciRxMsg[SPI_MAX_RX_MSG_LEN];
+static uint8_t __noinit g_hciRxMsg[SPI_MAX_RX_MSG_LEN];
```

Comment:
Not really related to this PR, but would be nice if someone could submit a patch to fix the naming of this variable, since it violates the Zephyr coding style. Using the same `rxmsg` name as `hci_spi_st.c` uses would be fine.

Reply:
@jhedberg, Is it OK to include in this PR? May separate as another commit.

Reply:
> @jhedberg, Is it OK to include in this PR? May separate as another commit.

Could be a second commit in this PR, or a new separate PR. I'm fine either way.